packages/features/auth/lib/next-auth-options.ts

```diff
@@ -100,9 +100,10 @@ const providers: Provider[] = [
         throw new Error(ErrorCode.InternalServerError);
       }
 
-      const user = await prisma.user.findUnique({
+      const user = await prisma.user.findFirst({
         where: {
           email: credentials.email.toLowerCase(),
+          linkedBy: null,
```

Comment:
As discussed, we will default to the parent user profile for authentication processes.